tests/ClientTest.php

```diff
@@ -10,21 +10,19 @@
 use PHPUnit\Framework\TestCase;
 use Sentry\ClientBuilder;
 use Sentry\Event;
-use Sentry\EventFactory;
 use Sentry\Options;
-use Sentry\Serializer\RepresentationSerializerInterface;
 use Sentry\Serializer\Serializer;
-use Sentry\Serializer\SerializerInterface;
 use Sentry\Severity;
 use Sentry\Stacktrace;
 use Sentry\Transport\HttpTransport;
+use Sentry\Transport\TransportFactoryInterface;
 use Sentry\Transport\TransportInterface;
 
 class ClientTest extends TestCase
 {
     public function testCaptureMessage(): void
     {
-        /** @var TransportInterface|MockObject $transport */
+        /** @var TransportInterface&MockObject $transport */
```

Comment:
With the proper PHPStan extension this annotation should be superfluous.

I will not repeat this comment on the other lines.

Reply:
It is, but the IDE is not able to recognize the full return type for the `createMock` function and the annotation helps it

Reply:
If that's the case (and you're using PHPStorm) you can use this plugin to make it work: https://plugins.jetbrains.com/plugin/9674-phpunit-enhancement/